schema/platform/resolvers/Mutation/assign.ts

```diff
@@ -113,8 +113,6 @@ async function assignWorkInstance(entity: string, to: ID, _ctx: Context) {
               SET
                   workresultinstancevalue = excluded.workresultinstancevalue,
                   workresultinstancemodifieddate = now()
-              WHERE
-                  nullif(wri.workresultinstancevalue, '') IS null
```

Comment:
My bad. This will still error if you try to reassign to the same worker. So this should be (I think);
```sql
WHERE
    nullif(wri.workresultinstancevalue, '') IS null
    OR wri.workresultinstancevalue != exclude.workresultinstancevalue
```